src/dmd/parse.d

```diff
@@ -2152,6 +2154,35 @@ final class Parser(AST) : Lexer
                             cppmangle = token.value == TOK.class_ ? CPPMANGLE.asClass : CPPMANGLE.asStruct;
                             nextToken();
                         }
+                        else if (token.value == TOK.string_) // extern(C++, "namespaces")
+                        {
+                            ignoreCppSymbols = true;
+                            idents = new AST.Identifiers();
+                            AST.StringExp stringExp = cast(AST.StringExp)parsePrimaryExp();
+                            char[] names = stringExp.toStringz();
```

Comment:
This is not the appropriate way to get a copy if you intend to alter it (and you need to remove const from the signature in order to do what you want).

Reply:
I was going to but all the functions in `Identifer` appear to assume that it has a null terminator.

https://github.com/dlang/dmd/blob/v2.082.0/src/dmd/identifier.d#L59

Including the strncmp, unless that `+ 1` is a bug.